test/torchaudio_unittest/torchscript_consistency_impl.py

```diff
@@ -624,3 +638,8 @@ def test_Vad(self):
         filepath = common_utils.get_asset_path("vad-go-mono-32000.wav")
         waveform, sample_rate = common_utils.load_wav(filepath)
         self._assert_consistency(T.Vad(sample_rate=sample_rate), waveform)
+
+    def test_SpectralCentroid(self):
+        filepath = common_utils.get_asset_path("vad-go-mono-32000.wav")
```

Comment:
Could you use either random Tensor or some synthetic wave here?
We are trying to minimize the use of external resource in test.

VAD is an exception here, as it requires audio with voice (although, since this is consistency test and not a an accuracy test, it does not need to use an actual voice, but that's a separate story).